pkg/scheduler/scheduler.go

```diff
@@ -599,14 +600,15 @@ func (sched *Scheduler) scheduleOne() {
 			ObjectMeta: metav1.ObjectMeta{Namespace: assumedPod.Namespace, Name: assumedPod.Name, UID: assumedPod.UID},
 			Target: v1.ObjectReference{
 				Kind: "Node",
-				Name: suggestedHost,
+				Name: scheduleResult.SuggestedHost,
 			},
 		})
 		metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
 		if err != nil {
 			klog.Errorf("error binding pod: %v", err)
 			metrics.PodScheduleErrors.Inc()
 		} else {
+			klog.V(3).Infof("pod gets bound successfully: %v/%v on node %v, evaluated %d nodes and %d feasible nodes", assumedPod.Namespace, assumedPod.Name, scheduleResult.SuggestedHost, scheduleResult.EvaluatedNodesNumber, scheduleResult.FeasibleNodesNumber)
```

Comment:
V(1) may be too verbose, but can we do this at V(2)? Sorry to nit-pick here, I'm just trying to make sure this log shows up at the right moment.

Reply:
Make sence.